values.yaml

```diff
@@ -20,10 +20,15 @@ kobotoolbox:
   #serviceAccountBackend: "redis://:pass@redis-master:6379/6"
   enketoApiKey: "change_me_please"
 
+# Actually post-install and pre-update
```

Comment:
Let's rename this on 4.x which could be when we merge the kobocat as an app work.